cdc/log.cc

```diff
@@ -838,6 +838,7 @@ class transformer final {
 
     api::timestamp_type _ts;
     bytes _tuuid;
+    lw_shared_ptr<cql3::untyped_result_set> _preimage_select_result;
```

Comment:
I'm not sure whether this is an improvement. Maybe reading the following patches will clear it up for me.

Reply:
Now I see that transformer willl need to have state now and it makes sense to store preimage in it.